css/amp.css

```diff
@@ -334,3 +334,13 @@ amp-analytics {
   overflow: hidden !important;
   visibility: hidden;
 }
+
+amp-user-notification {
+  position: fixed !important;
```

Comment:
ill move. was trying to figure out the style issue but should be fixed in your new PR.